deployment/helm/synopsys-alert/templates/alert-environ-configmap.yaml

```diff
@@ -1,13 +1,20 @@
 apiVersion: v1
 kind: ConfigMap
 data:
+  {{- if .Values.alert.hostname }}
+  ALERT_HOSTNAME: {{ .Values.alert.hostname | quote }}
+  {{- else }}
+  ALERT_HOSTNAME: {{ .Release.Name | quote }}
+  {{- end}}
+  {{- if .Values.alert.port }}
+  ALERT_SERVER_PORT: {{ .Values.alert.port | quote }}
+  {{- else }}
```

Comment:
@psantos1113 can you pls remove if and else for ALERT_SERVER_PORT and set them to values.alert.port always? Since the default is 8443, we don't need if-else